### ISSUE TITLE
[BUG] Lost Cousin load failures - 3 - US 1940 Census references incomplete

### ISSUE THREAD

**Issue by bmurcutt** [ShammyLevva/FTAnalyzer#105]
Version 7.3.7.1

The following scenario failed to load:

US Federal Census 1940 - Failed to add I6822: Marilyn Reed b.16 SEP 1925,   Roll:  4515 BURLINGTON, ED: 51-5, Page:   4A. Census Reference problem.

Manually trying to load on LC identified that LC is expecting a Sheet value of 4A, not a Page value.

The GEDCOM census reference uses the term 'Sheet', as does LC.  Presumably one of the 1940 patterns works (I haven't figured which yet) to have gotten this far, but it looks like the Page/Sheet reference is not being handled correctly when trying to load to LC

GEDCOM details:

1 EVEN Route 2, Honey Creek Road
2 TYPE Census-Household Member
2 DATE 1940
2 PLAC Burlington, Racine County, Wisconsin, USA
2 NOTE Role: Household Member
3 CONT
3 CONT «b»«i»First name(s)    Last name    Relationship    Marita
3 CONC l Status    Gender    Age    Birth year    Birth place
3 CONT Alice    Reed    Wife    Married    Female    49    189
3 CONC 1    Wisconsin
3 CONT Clifford    Reed    Head    Married    Male    50    189
3 CONC 0    Wisconsin
3 CONT «/b»Donald    Reed    Son    Married    Male    22    191
3 CONC 8    Wisconsin
3 CONT Marilyn    Reed    Daughter    Single    Female    15    19
3 CONC 25    Wisconsin
3 CONT «b»  «/b»
3 CONT «b»Ethnicity   «/b»American
3 CONT «b»Race   «/b»White
3 CONT «b»City/township   «/b»Burlington
3 CONT «b»County   «/b»Racine
3 CONT «b»State   «/b»Wisconsin
3 CONT «b»Previous residence date   «/b»1935
3 CONT «b»Previous residence city   «/b»Same Hous
3 CONC e
3 CONT «b»Previous residence county   «/b»Racin
3 CONC e
3 CONT «b»NARA series   «/b»T627
3 CONT «b»NARA roll   «/b»4515
3 CONT «b»Record set   «/b»Us Census 1940
3 CONT «b»Category   «/b»Census, land & surveys
3 CONT «b»Subcategory   «/b»Census
3 CONT «b»Collections from   «/b»Americas, United State
3 CONC s
3 CONT
2 SOUR @S360@
**3 PAGE T627, roll 4515, Burlington, enumeration district (ED) 51-5
4 CONC , sheet 4A,  household 45, Alice Reed, accessed 22 Jan 2019**
3 QUAY 3
3 DATA
4 TEXT
5 CONT First name(s)    Last name    Relationship    Marital Statu
5 CONC s    Gender    Age    Birth year    Birth place
5 CONT Alice    Reed    Wife    Married    Female    49    189
5 CONC 1    Wisconsin
5 CONT Clifford    Reed    Head    Married    Male    50    189
5 CONC 0    Wisconsin
5 CONT Donald    Reed    Son    Married    Male    22    1918    W
5 CONC isconsin
5 CONT Marilyn    Reed    Daughter    Single    Female    15    19
5 CONC 25    Wisconsin
5 CONT
5 CONT Ethnicity   American
5 CONT Race   White
5 CONT City/township   Burlington
5 CONT County   Racine
5 CONT State   Wisconsin
5 CONT Previous residence date   1935
5 CONT Previous residence city   Same House
5 CONT Previous residence county   Racine
5 CONT NARA series   T627
5 CONT NARA roll   4515
5 CONT Record set   Us Census 1940
5 CONT Category   Census, land & surveys
5 CONT Subcategory   Census
5 CONT Collections from   Americas, United States
3 OBJE
4 FORM jpg
4 FILE C:\Users\BrianM\Documents\Family Tree\Records\Hey records\Alice Grace Hocking 1891 - Census 1940 US.jpg
4 _DATE 1940
4 _SCBK Y
4 _PRIM Y
4 _TYPE PHOTO

**Comment by ShammyLevva** [comment]
Yup not really looked at US census refs in detail yet. Probably lots of scope for adding extra patterns.

**Comment by bmurcutt** [comment]
I only have 20-odd to date.

And the reference details needed aren't all in the (FindMyPast) transcript, so you have to find them in the image itself!  It's a new area for me, so it will be inconsistent and not always correct!

They’re all in the 'failed' file I sent through too I think 😀
